app/controllers/keywords_controller.rb

```diff
@@ -2,4 +2,19 @@
 
 class KeywordsController < ApplicationController
   def index; end
+
+  def create
```

Comment:
@malparty As per our new discussion, do we need this action. Will it be a feature used by the end user of the application?

Reply:
@ankitkalia1195  Nope, I used it a lot to preview the results and later to build the parser, but after getting used to vcr cassette and using pre-recorded html content, it's indeed useless. 
I've just removed it (as well as the form in the index view).